v63003/u_inref/gtm8767.csh

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+set randint = $$
+set randhex = `openssl rand -hex 3`
+set randnum = $$
+
+$gtm_tst/com/dbcreate.csh mumps 1 >>& create1.out
+echo "Setting Hard Spin Count"
+$MUPIP SET -region DEFAULT -HARD_SPIN_COUNT=$randint
+#Will only display if Hard Spin Count is correct in DSE Dump
```

Comment:
Remove this line and instead add a line like

echo "# Verify the random hard spin count shows up in DSE DUMP -FILE"

Also do similar change in other places where DSE DUMP -FILE is invoked.